tools/rexm/rexm.c

```diff
@@ -436,7 +446,7 @@ int main(int argc, char *argv[])
             SaveFileText(TextFormat("%s/%s/%s.c", exBasePath, exCategory, exName), exTextUpdated[1]);
             for (int i = 0; i < 6; i++) { MemFree(exTextUpdated[i]); exTextUpdated[i] = NULL; }
             UnloadFileText(exText);
-        }
+        } break;
```

Comment:
This break shouldn't be here, when `create` a new example it should also be `added`